test/suite.sh

```diff
@@ -13,6 +13,7 @@ assert "node bin/cmd.js < test/async-long.js" "1
 3
 4
 5"
+assert_end basic
```

Comment:
I didn't know about this feature. Very nice with a test suite that takes this long to run. Thanks!